app/code/Magento/Sales/view/adminhtml/templates/items/column/qty.phtml

```diff
@@ -4,35 +4,42 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php if ($item = $block->getItem()) : ?>
+<?php if ($item = $block->getItem()): ?>
     <table class="qty-table">
         <tr>
             <th><?= $block->escapeHtml(__('Ordered')); ?></th>
             <td><?= (float) $item->getQtyOrdered() ?></td>
         </tr>
 
-        <?php if ((float)$item->getQtyInvoiced()) : ?>
+        <?php if ((float)$item->getQtyBackordered()): ?>
+            <tr>
+                <th><?= $block->escapeHtml(__('Backordered')); ?></th>
+                <td><?= (float) $item->getQtyBackordered() ?></td>
```

Comment:
Using `escape` functions from the Template class was deprecated. Please use `$escaper`.
It would be good replace all calls in this file.